src/pages/sc-cycle-counts/Forms/Controller.js

```diff
@@ -0,0 +1,177 @@
+import { useContext, useEffect } from 'react'
+import { useForm } from 'src/hooks/form'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { SCRepository } from 'src/repositories/SCRepository'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import toast from 'react-hot-toast'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import FormShell from 'src/components/Shared/FormShell'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+
+const Controller = ({ store, maxAccess, labels }) => {
+  const { recordId, isPosted, isClosed } = store
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const { formik } = useForm({
+    maxAccess,
+    enableReinitialize: true,
+    validateOnChange: true,
+    initialValues: {
+      stockCountId: '',
+      rows: []
+    },
+    onSubmit: async () => {
+      try {
+        const itemsList = formik.values.rows
+        .map((item, index) => ({
+          ...item,
+          id: index + 1,
+          status: item.status || 1,
+          controllerId: item.recordId,
+          siteId: formik.values.siteId,
+          stockCountId: recordId,
+        }))
+        .filter(item => item.isChecked); 
+
+        const data = {
+          stockCountId: recordId,
+          siteId: formik.values.siteId,
+          items: itemsList
+        }
+
+        await postRequest({
+          extension: SCRepository.PHY.set2,
+          record: JSON.stringify(data)
+        })
+        toast.success(platformLabels.Updated)
+        const res2 = await fetchGridData(recordId, formik.values.siteId)
```

Comment:
why u are passing formik.values.siteId? cannot be used directly inside function?